conflict-resolution.html

```diff
@@ -0,0 +1,108 @@
+    decision is made by someone else who's empowered to make it.
+</p>
+
+<p>The <a href="#sync-protocol">synchronization protocol</a> between parties to a DID doc allows them to have
+    different opinions about order, and to adjust that order over time, based on evidence. A delete delta
+    "bubbles" to a position in history that's after the last delta that would have been impossible before it,
+    AND that was accepted by any party that had to make authorization decisions. A delete delta that was never
+    accepted by anyone (as with one of the mutually contradictory deltas above) just disappears. And deltas that
+    were discarded by a party because of ordering differences will get re-introduced to history once evidence is
+    found that a party making authorization decisions saw the order differently. This is essentially like the
+    Republic of Bob official saying, "For now, I'm going to pretend that Foo's appointment request is rejected,
+    since I saw it after the notice that Foo was fired. But if I see that my boss has scheduled the appointment,
+    I'll put the appointment request into my official log of Acmeland actions in the other order, because my
+    boss must have received the letters in the other order."
+</p>
+
+<p>The net effect of these two rules for deletes is that different parties can have different views of where
+    a delete fits into history, and those views will converge over time, <em>to the extent that convergence
+        matters</em>. Where convergence never reaches closure, that means there's no evidence that forces one of
+    the views to be more correct than the other. And that's okay.
+</p>
+
+<p>You may be saying, "Yes, but what if the ambassador was fired in the morning, and I can prove it, and
+    someone still gave the ambassador privileges twelve hours later in the evening?" Or what if one party
+    accepted <code>delta[0]</code>, and another party accepted <code>delta[1]</code> (that were mutually
+    contradictory)? How do those get reconciled?
+</p>
+
+<p>They don't, and it doesn't matter. In the first case, the delete's timing moves to the evening in history;
+    that's the true last time that the ambassador's privileges had any effect, even if the intent was to
+    de-authorize earlier. The deltas are not an expression of intention, but an expression of when that intention
+    was honored in any way that mattered. In the second case, the contradictory views of reality are a real
+    historical event, and the fact that they caused inconsistent authorizations to happen in different corners
+    of the world isn't going to be reconciled into an explanation that pretends otherwise. The deltas on top
+    of the genesis doc end up containing both deletes, as well as some deltas inconsistent with both deletes.
+    At least once both deletes have been applied, the potential for new inconsistencies collapses. Real life in
+    a decentralized world is messy like this.
+</p>
```

Comment:
I think this section says "Updates are processed in the order received. It's up to the writing agent to manage the changes they allow to prevent malicious behaviour. Further, key rotations are handled as delete/update operations.".  Is there more from a spec perspective?

I think the narrative distracts from the spec. If it is needed, it needs to be separate from the details that are needed.

Reply:
This text - while illustrative - distracts from the declared intention of this PR. It's a separate topic of its own that requires some discussion, imo.

Reply:
@swcurran : I think your summary is close, but a bit too simple. A number of people I've talked to about this spec have concerns that need to be dispelled. Your summary, while approximately true, is probably not quite enough to satisfy them; at least, it hasn't been when I've tried to use it verbally. But I take your point, which is that this stuff bogs down the spec. I haven't found a good way to release collateral (supporting docs) for the spec, which is why some of this material has no better home.

For now, I've deleted the final two long paragraphs of the section, which is where the rat hole feels especially deep. I can bring them back in a separate PR with separate discussion, as @llorllale suggested.